packages/compiler/server/serverlib.ts

```diff
@@ -645,11 +645,11 @@ export function createServer(host: ServerHost): Server {
     const nodevalueDir = hasTrailingDirectorySeparator(node.value)
       ? node.value
       : getDirectoryPath(node.value);
-    const maincadl = resolvePath(documentDir, nodevalueDir);
-    const listFiles = (await program.host.readDir(maincadl)).filter(
+    const mainCadl = resolvePath(documentDir, nodevalueDir);
+    const Files = (await program.host.readDir(mainCadl)).filter(
```

Comment:
sorry to keep nitpicking on the names, but it should be `files` with a small `f`